src/queue.cpp

```diff
@@ -1041,8 +1041,8 @@ cvk_command_kernel::build_batchable_inner(cvk_command_buffer& command_buffer) {
     // Setup printf buffer descriptor if needed
     if (m_kernel->program()->uses_printf()) {
         // Create and initialize the printf buffer
-        auto buffer = m_queue->get_or_create_printf_buffer();
-        auto err = m_queue->reset_printf_buffer();
+        auto buffer = get_or_create_printf_buffer();
+        auto err = reset_printf_buffer();
```

Comment:
I don't think we will ever reuse that buffer.
Those 2 functions could be merged into a single `create_printf_buffer`